### PR TITLE
Story/vspc 146

### PR DESCRIPTION
**Pull request by prjadhav14**
PR to Compare and merge Staff Module fixes with develop branch

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
- Elements should be aligned
image
- dropdown and buttons should be smaller
- date in info box should be formatted according to user's timezone (moments.js will probably help with that https://momentjs.com/)

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html [hunk 1]
```diff
@@ -64,6 +65,19 @@
...
+    
+    var dateFormat = 'YYYY-DD-MM HH:mm:ss';
+    var data = [[${module.creationDate}]];
+    var month = data.monthValue<10 ? "0"+data.monthValue : data.monthValue;
+    var day = data.dayOfMonth<10 ? "0"+data.dayOfMonth : data.dayOfMonth;
+    var hour = data.hour<10 ? "0"+data.hour : data.hour;
+    var minute = data.minute<10 ? "0"+data.minute : data.minute;
+    var second = data.second<10 ? "0"+data.second : data.second;
+    var date = data.year+"-"+month+"-"+day+" "+hour+":"+minute+":"+second;
+    var testDateUtc = moment.utc(date);
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html:77]: what does the creation date look like that you need to do all of this? moments should do the formatting for you.

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html [hunk 2]
```diff
@@ -93,6 +107,7 @@
 	
 	$(function () {
 		$('[data-toggle="tooltip"]').tooltip()
+		$('[data-toggle="tooltip"]').title="Prashant Hello"
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html:110]: Hi Prashant

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
besides that should be good to go

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html (hunk 1 above)
Comment (prjadhav14) [reply vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html:77]: The creationDate is read as date object in the javascript.
{
  "offset": {
    "totalSeconds": -25200,
    "id": "-07:00",
    "rules": {
      "fixedOffset": true,
      "transitions": [],
      "transitionRules": []
    }
  },
  "monthValue": 6,
  "year": 2020,
  "month": "JUNE",
  "dayOfMonth": 30,
  "dayOfWeek": "TUESDAY",
  "dayOfYear": 182,
  "nano": 504000000,
  "hour": 11,
  "minute": 25,
  "second": 0
}
I could not find the function in the moment library to format the date object.
Any leads, please?

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html (hunk 2 above)
Comment (prjadhav14) [reply vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html:110]: Opps, deleted the loc. :-P

File: vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html (hunk 1 above)
Comment (jdamerow) [reply vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html:77]: Have you tried reading it as a string in js? I believe it already comes in a format that moments can read.

**Pull request merged**